src/Sylius/Behat/spec/Context/Setup/PromotionContextSpec.php

```diff
@@ -159,4 +160,29 @@ function it_creates_percentage_shipping_discount_action_for_promotion(
 
         $this->itGivesPercentageDiscountOnShippingToEveryOrder($promotion, 0.1);
     }
+
+    function it_creates_item_percentage_discount_action_for_promotion_products_with_specific_taxon(
+        $actionFactory,
```

Comment:
I never heard about this convention, I recall we discussed to add these everywhere to have proper autocomplete and be consistent. Not a big deal, but please use that. It actually does not matter if deps is injected via constructor or passed as argument. I don't see any benefit of having that distinction. These are all collaborators.